web/portal/src/main/webapp/WEB-INF/conf/sso/oauth-configuration.xml

```diff
@@ -129,6 +129,50 @@
       </value-param>
     </init-params>
   </component>
+  
+  <component>
+    <key>org.gatein.security.oauth.openid.OpenIdProcessor</key>
+    <type>org.gatein.security.oauth.openid.OpenIdProcessorImpl</type>
+    <init-params>
+      <value-param>
+        <name>clientId</name>
+        <value>${gatein.oauth.openid.clientId}</value>
```

Comment:
IMO we need to use the pattern exo.oauth.openid instead of gatein.oauth.openid

Reply:
Done. In addition in documentation I put exo.